### ISSUE TITLE
Add line numbers to the Threads & Variables panel

### ISSUE THREAD

**Issue by Ashark** [Camelcade/Devel-Camelcadedb#64]
Currently (Perl plugin 233.9102.97-EAP-SNAPSHOT), when debugging, in "Threads & Variables" panel, you can see stack lines. But they do not have line numbers at which the line is currently staying.
However, the debugger does apparently know that numbers, because it jumps there when I click.
It would be nice to see line numbers (like it is expected for any other language, for example for python in PyCharm).

<img src=https://github.com/Camelcade/Devel-Camelcadedb/assets/22634975/5b7c0c27-b022-41f3-aa0c-3d8d12a9dd47 width=50% />

<img src=https://github.com/Camelcade/Devel-Camelcadedb/assets/22634975/11501d7f-1a75-481b-94cd-c833f2bc7ed3 width=50% />

**Comment by hurricup** [comment]
Yep, good point thank you. However it is possibly request to IDE part, not debugger backend. I'll investigate.

**Comment by hurricup** [comment]
Yep. 

![image](https://github.com/Camelcade/Devel-Camelcadedb/assets/2811330/1d38c05b-2d42-4a85-b0bd-7c60c88464ae)
